airbyte-integrations/connectors/source-ebay-finance/manifest.yaml

```diff
@@ -0,0 +1,471 @@
+version: 6.41.5
+
+type: DeclarativeSource
+
+description: |-
+  Website: https://www.ebay.com/
+  Documentation: https://developer.ebay.com/api-docs/sell/finances/overview.html
```

Comment:
PR Fix as suggested in [review here](https://github.com/airbytehq/airbyte/pull/56920#pullrequestreview-2730204430): 

Adds link to website, and docs. The data we get from is from ebay sandbox API. Which I used as well for testing this connector in connector builder.